test/test_linalg.py

```diff
@@ -1910,8 +1909,8 @@ def is_broken_matrix_norm_case(ord, x):
                 if is_broken_matrix_norm_case(ord, x):
                     continue
                 else:
-                    result = torch.linalg.norm(x, ord=ord)
                     result_n = np.linalg.norm(x_n, ord=ord)
+                    result = torch.linalg.norm(x, ord=ord)
```

Comment:
Reordered here so that whoever tries to enable skipped matrix_ords in the future first sees the failures from NumPy.